securedrop/tests/test_crypto_util.py

```diff
@@ -23,3 +24,23 @@ def test_clean(self):
             crypto_util.clean('foo bar`') # backtick is not currently allowed
         with self.assertRaises(crypto_util.CryptoException):
             crypto_util.clean('bar baz~') # tilde is not currently allowed
+
+    def test_encrypt(self):
+        journalist, _ = utils.db_helper.init_journalist()
```

Comment:
You can remove this `init_journalist()` line as you should only need 

```
source, _ = utils.db_helper.init_source()
```

for the setup for the function under test here